homeassistant/components/goalzero/strings.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "Goal Zero Yeti",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Goal Zero Yeti",
+        "data": {
+          "host": "host",
```

Use pre-defined key values:

```suggestion
          "host": "[%key::common::config_flow::data::host%]",
```